test/data/test_builtin_datasets.py

```diff
@@ -18,13 +18,12 @@ def conditional_remove(f):
 
 
 class TestDataset(TorchtextTestCase):
+    def helper_test_func(self, length, target_length, results, target_results):
```

Comment:
Using the [style guide](https://www.python.org/dev/peps/pep-0008/#designing-for-inheritance) we should probably prefix this with a leading underscore.

Reply:
As a suggestion, you could have one helper for datasets and one for iterators if you want to distinguish between these two cases to make writing code easier. You could also move the tensor construction ```torch.tensor([...], dtype=torch.int64)``` into these helpers.